cmp/53fe1e2b-6a1e-4619-8da2-be8f3b9ab267/component.lp

```diff
@@ -0,0 +1,30 @@
+(librepcb_component 53fe1e2b-6a1e-4619-8da2-be8f3b9ab267
+ (name "Thermistor")
+ (description "")
+ (keywords "")
+ (author "J. R. Schmid")
+ (version "0.1")
+ (created 2020-05-10T13:35:28Z)
+ (deprecated false)
+ (category 1039f038-20a6-4bfe-89c1-99f34fbb45bd)
+ (schematic_only false)
+ (default_value "{{PARTNUMBER or DEVICE}}")
+ (prefix "TH")
```

Comment:
According to IEEE, the designator for a thermistor would be "RT". Is "TH" widely used?

Reply:
**Device:**

> I would tend towards not including generic thermistors at all in the base library since there are so many different packages available on the market, while at the same time thermistors are way less common than resistors.

Okay, so I'll move the *device* to another library (which one?), open a new PR there, and force-push to this one with the device removed?

**Symbol:**

> Naming: If I interpret the arrows correctly, then this is a NTC resistor, right? The symbol name should reflect that. Also, since this is the IEC (EU) variant of the symbol and there is also an IEEE (US) variant, the name should end with " EU".

Changed to `Thermistor EU`.

> Appearance: I would adjust the symbol a bit to match the IEC standard.

The appearance was copied 1:1 from KiCAD. Is it, generally speaking, a bad idea to look there for inspiration?

**Component:**

> According to IEEE, the designator for a thermistor would be "RT". Is "TH" widely used?

That was from KiCAD I believe (?) Or if not, it's what gf's professor used. IEEE sounds fine though; changed it to RT.

Reply:
> Is it, generally speaking, a bad idea to look there for inspiration?

No, but in general if there's a standardized symbol, I'd always follow that.

> Okay, so I'll move the device to another library (which one?), open a new PR there, and force-push to this one with the device removed?

Yep! I'd simply move it to the manufacturer-specific library with the appropriate part name.

Reply:
> Yep! I'd simply move it to the manufacturer-specific library with the appropriate part name.

Could you create one for muRata?

![image](https://www.mouser.de/images/suppliers/90px/murataelectronics.png)

Reply:
Done! https://github.com/LibrePCB-Libraries/Murata.lplib